yjit/src/core.rs

```diff
@@ -322,6 +322,133 @@ pub struct Context {
     temp_mapping: [TempMapping; MAX_TEMP_TYPES],
 }
 
+// Compressed version of Context
+type PackedContext = Box<[ContextDelta]>;
```

Comment:
Is this a box containing a vec?

Reply:
I believe it's a box containing a [slice](https://doc.rust-lang.org/std/slice/index.html). A vec seems to use 24 extra bytes besides the vector content, and this seems to use 16 extra bytes the slice content (8 bytes for a Box pointer, and another 8 for ???).